core/federated/RTI/rti_remote.c

```diff
@@ -1265,9 +1139,15 @@ static int32_t receive_and_check_fed_id_message(int* socket_id, struct sockaddr_
   // The IP address is stored here as an in_addr struct (in .server_ip_addr) that can be useful
   // to create sockets and can be efficiently sent over the network.
   // First, convert the sockaddr structure into a sockaddr_in that contains an internet address.
-  struct sockaddr_in* pV4_addr = client_fd;
+  // struct sockaddr_in* pV4_addr = client_fd;
```

Comment:
Why is this commented out?

Reply:
Oh, I should probably remove this.

Actually this part will be removed in the future. This part will be included in the accept() function, when in the next PR.

This part was refactored to remove the input argument `struct sockaddr_in* client_fd` in this function.